File: Frontend-Kaikan-Restobar/src/app/core/models/categoria.ts
export class Categoria{
    id: number | null;
    nombre: string;
    img_url:string;

    constructor(id: number | null, nombre: string, img_url:string){
        this.id = id;
        this.nombre = nombre;
        this.img_url=img_url;
    }
}

File: BackendKaikan/src/main/java/com/kaikan/model/categoria/Categoria.java
package com.kaikan.model.categoria;


import com.kaikan.generics.BaseEntity;
import jakarta.persistence.Entity;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Data
@Table(name = "categoria", schema = "public")
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter

public class Categoria extends BaseEntity {
    private String nombre;
    private String img_url;
}
